client/foundation/foundation_api.go

```diff
@@ -45,6 +48,9 @@ func NewFoundationAPIClient(credentials client.Credentials) (*Client, error) {
 		FileManagement: FileManagementOperations{
 			client: client,
 		},
+		Networking: NetworkingOperations{
+			client: client,
```

Comment:
Make sure that client is not writing to any global variable. As we are using a single client object in all foundation service's clients it may cause race condition in parallel execution or cause unwanted behaviour.

Reply:
Yes @siddharth-kulshrestha . Parallel datasource/resource running can cause issues ?

Reply:
If yes we need to do something about it